docs/tutorial/part-five/index.md

```diff
@@ -75,47 +75,55 @@ module.exports = {
+Klik dropdown `allFile`. Posisikan kursor Anda setelah `allFile` di area permintaan, dan lalu ketik <kbd>Ctrl + Enter</kbd>. Ini akan mengisi `id` untuk setiap file terlebih dahulu. Tekan "Mainkan" untuk menjalankan kueri:
 
 ![filesystem-query](filesystem-query.png)
 
-In the Explorer pane, the `id` field has automatically been selected. Make selections for more fields by checking the field's corresponding checkbox. Press "Play" to run the query again, with the new fields:
+Di panel Exploere, bidang `id` telah dipilih secara otomatis. Tentukan pilihan untuk lebih banyak bidang dengan mencentang kotak yang sesuai pada bidang tersebut. Tekan "Mainkan" untuk menjalankan kueri lagi, dengan bidang baru:
 
-![filesystem-explorer-options](filesystem-explorer-options.png)
+![pilihan-pada-filesystem-explorer](filesystem-explorer-options.png)
 
-Alternatively, you can add fields by using the autocomplete shortcut (<kbd>Ctrl + Space</kbd>). This will show queryable fields on the `File` nodes.
+Alternatifnya, Anda dapat menambahkan bidang dengan menggunakan pintasan _autocomplete_ (<kbd>Ctrl + Space</kbd>). Ini akan menampilkan bindang yang dapat diminta pada node `File`.
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
-each time to re-run the query. You'll see the updated query results:
+Coba tambahkan sejumlah bidang ke kueri Anda, dengan menekan <kbd>Ctrl + Enter</kbd>
+setiap kali menjalankan kueri kembali. Anda akan melihat hasil _query_ yang diperbarui:
 
 ![allfile-query](allfile-query.png)
 
-The result is an array of `File` "nodes" (node is a fancy name for an object in a
-"graph"). Each `File` node object has the fields you queried for.
+Hasilnya adalah array dari `File` "nodes" (_node_ adalah nama mewah untuk objek di "_graph_"). Setiap objek `File` _node_ memiliki bidang yang Anda minta.
 
-## Build a page with a GraphQL query
+## Membuat halaman dengan GraphQL _query_
+
+Membangun halaman baru dengna Gatsby sering dimulai dengan GraphiQL. Anda pertama-tama membuat sketsa
+kueri data dengan bermain di GraphiQL lalu salin ini ke halaman komponen React
+untuk mulai membangun UI.
+
+Ayo Coba ini.
 
 Building new pages with Gatsby often starts in GraphiQL. You first sketch out
 the data query by playing in GraphiQL then copy this to a React page component
 to start building the UI.
 
 Let's try this.
 
+Buat file baru di `src/pages/my-files.js` dengan `allFile` GraphQL _query_ Anda baru saja
+dibuat
+
 Create a new file at `src/pages/my-files.js` with the `allFile` GraphQL query you just
 created:
 
 ```jsx:title=src/pages/my-files.js
-import React from "react"
-import { graphql } from "gatsby"
-import Layout from "../components/layout"
+import React from 'react'
```

Comment:
Please dont change any code for line 124- 126

Reply:
i not change any code and I do not know why this line changed, but it will be fixed